tests/queries/0_stateless/01702_system_query_log.reference

```diff
@@ -42,7 +42,7 @@ Alter	ALTER TABLE sqllt.table DROP COLUMN the_new_col;
 Alter	ALTER TABLE sqllt.table UPDATE i = i + 1 WHERE 1;
 Alter	ALTER TABLE sqllt.table DELETE WHERE i > 65535;
 Select	-- not done, seems to hard, so I\'ve skipped queries of ALTER-X, where X is:\n-- PARTITION\n-- ORDER BY\n-- SAMPLE BY\n-- INDEX\n-- CONSTRAINT\n-- TTL\n-- USER\n-- QUOTA\n-- ROLE\n-- ROW POLICY\n-- SETTINGS PROFILE\n\nSELECT \'SYSTEM queries\';
-System	SYSTEM FLUSH LOGS
+System	SYSTEM FLUSH LOGS;
 System	SYSTEM STOP MERGES sqllt.table
```

Comment:
But why there's a semicolon now? And why nothing changed for other SYSTEM queries?

Reply:
This one is weird and I need to look again at why.

Reply:
Because some types of SYSTEM queries end with ON CLUSTER, after parsing the type, it will continue to parse ON CLUSTER, so if query without ON CLUSTER clause, tokens will iterate to the semicolon. For other SYSTEM queries without semicolons, after parsing the ON CLUSTER, there are some Identifiers behind them. After parsing these Identifiers, parsing will not continue, and tokens will not iterate to the last semicolon.